.travis.yml

```diff
@@ -23,72 +28,42 @@ notifications:
+  - "python requirements.py --extras=$REXTRAS --level=dev > .travis-devel-requirements.txt"
 
 install:
-  - "sudo apt-get -qy install apache2 libapache2-mod-wsgi libapache2-mod-xsendfile ssl-cert poppler-utils git subversion nodejs --fix-missing"
-  - "sudo a2enmod actions"
-  - "sudo a2enmod version || echo ':('"
-  - "sudo a2enmod rewrite"
-  - "sudo mkdir /etc/apache2/ssl"
-  - "sudo /usr/sbin/make-ssl-cert generate-default-snakeoil /etc/apache2/ssl/apache.pem"
   - "travis_retry pip install unittest2"
-  - "travis_retry pip install -e git+https://github.com/inveniosoftware/invenio#egg=invenio"
-  - "travis_retry pip install -e .[docs] --quiet"
-  # - "python setup.py compile_catalog"
-  - "inveniomanage config create secret-key"
-  - "inveniomanage config set CFG_EMAIL_BACKEND flask_email.backends.console.Mail"
-  - "inveniomanage config set CFG_BIBSCHED_PROCESS_USER `whoami`"
-  - "inveniomanage config set PACKAGES ['invenio.base', 'invenio.modules.accounts', 'invenio.modules.records', 'invenio_tags']"
-  - "inveniomanage config set CFG_TMPDIR /tmp"
-  # - "sudo su -c \"npm update\""
-  # - "sudo su -c \"npm install --silent -g bower less clean-css uglify-js requirejs\""
-  # All the step below this points are solely for test purposes, don't use them
-  # to setup your invenio installation. Please do RTFM instead (INSTALL.rst).
-  # - "inveniomanage bower -i bower-base.json > bower.json"
-  # - "bower install --silent"
-  # - "inveniomanage config set COLLECT_STORAGE flask_collect.storage.link"
-  # - "inveniomanage collect > /dev/null"
-  # - "inveniomanage assets build"
-  - "inveniomanage config set CLEANCSS_BIN false"  # deactivate all the things
-  - "inveniomanage config set LESS_BIN false"      # false is /usr/bin/false
-  - "inveniomanage config set REQUIREJS_BIN false"
-  - "inveniomanage config set UGLIFYJS_BIN false"
-  - "inveniomanage config set ASSETS_AUTO_BUILD False"
+  - "travis_retry pip install -r .travis-$REQUIREMENTS-requirements.txt --allow-all-external --quiet"
+  - "travis_retry pip install -e .[$REXTRAS] --quiet --process-dependency-links"
 
 before_script:
-  # - "inveniomanage apache create-config"
-  # - "sudo ln -s $VIRTUAL_ENV/var/invenio.base-instance/apache/invenio-apache-vhost.conf /etc/apache2/sites-enabled/invenio.conf"
-  # - "sudo ln -s $VIRTUAL_ENV/var/invenio.base-instance/apache/invenio-apache-vhost-ssl.conf /etc/apache2/sites-enabled/invenio-ssl.conf"
-  # - "sudo /usr/sbin/a2dissite *default* || echo ':('"
-  # - "sudo /usr/sbin/a2enmod ssl" # enable SSL module
-  # - "sudo apachectl configtest && sudo service apache2 restart || echo 'Apache failed ...'"
   - "inveniomanage database init --yes-i-know || echo ':('"
   - "inveniomanage database create --quiet || echo ':('"
-#  - "inveniomanage demosite populate --yes-i-know"
 
 script:
-  - "sphinx-build -qnNW docs docs/_build/html"
+  - "sphinx-build -qnN docs docs/_build/html"
```

Comment:
Nope, it does not work then, because sphinx produces a warning when you include external graphics in your documentation. We do this by the usage of "badges", which appear nicely on the main/landing page of the docs. If you find a method to disable this specific kind of warning, go ahead. I was not able to find something useful and it seems to be the behaviour that is wanted by the sphinx devs.